.gitignore

```diff
@@ -4,3 +4,4 @@ __pycache__
 .mypy_cache
 .pytest_cache
 .flake8
+.vscode
```

Comment:
It would seem safer to add it to your global .gitignore if VSCode is your editor of choice.  I believe @dirk-thomas uses VSCode, is that the approach that you use?

Reply:
Since there is an almost infinite amount of possible editors and custom config files, yes, I think those should be ignored by the user globally.

Reply:
removed